packages/hydrogen-react/src/HydrogenProvider.example.jsx

```diff
@@ -0,0 +1,20 @@
+import {HydrogenProvider, useShop} from '@shopify/hydrogen-react';
+
+export default function App() {
+  return (
+    <HydrogenProvider countryIsoCode="CA" languageIsoCode="EN">
```

Comment:
Nit picky, but we should be consistent on what these are called throughout Hydrogen:

1. https://github.com/Shopify/hydrogen/blob/main/templates/skeleton/app/lib/context.ts#L33
2. https://shopify.dev/docs/api/hydrogen/2025-01/utilities/getshopanalytics#getShopAnalytics-returns